src/System.Private.ServiceModel/tools/scripts/WCFCustomCleanup.cmd

```diff
@@ -0,0 +1,36 @@
+@if "%_echo%" neq "on" echo off
```

Comment:
Who do we expect to be calling WcfCustomCleanup.cmd and WcfCustomSetup,cmd? It seems like you've assumed that this will be called only via MSBuild during a full build. If this is the case, I recommend either renaming the script to something that describes it (say, WcfServiceMSBuildCleanup.cmd, WcfServiceMSBuildSetup.cmd) or do some kind of check at the tops of the scripts to check it's running in the context of MSBuild. 

Not blocking - but as I read through the script it's apparent to me that you've assumed certain conditions for this script to work (e.g., that the CertificateGenerator is around)